apps/teams-test-app/src/components/LocationAPIs.tsx

```diff
@@ -69,11 +69,33 @@ const ShowLocation = (): React.ReactElement =>
     },
   });
 
+const HasGeoLocationPermission = (): React.ReactElement =>
```

Comment:
Potential nit: I think most of our UI components put the Has[InsertNameHere]Permission as the last UI element in the section. I would have to confirm this, but maybe we want to keep that consistent pattern?